src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -3248,24 +3117,24 @@ public synchronized InputStream getInputStream(Range range, RendererConfiguratio
 
 		// Pipe transcoding result
 		OutputParams params = new OutputParams(configurationSpecificToRenderer);
-		params.aid = getMediaAudio();
-		params.sid = media_subtitle;
-		params.header = getHeaders();
-		params.mediaRenderer = mediarenderer;
+		params.setAid(getMediaAudio());
+		params.setSid(media_subtitle);
+		params.setHeader(getHeaders());
```

Comment:
The setter and getter don't match - shouldn't it be either `setHeaders` and `getHeaders`, or `setHeader` and `getHeader`?

Reply:
@SubJunk it seems like plugins implementation whose are no longer supported.

Reply:
I think we can do that, but let's do it after this branch merges?

Reply:
@SubJunk because the `getHeaders()` is used only locally and is alvays `null` we can remove it now.